packages/manager-components/src/hooks/useCatalogPrice.ts

```diff
@@ -19,7 +19,14 @@ const FRENCH_FORMAT = [
   'TN',
 ];
 
-export const useCatalogPrice = (maximumFractionDigits?: number) => {
+export interface CatalogPriceOptions {
```

Comment:
```suggestion
export type CatalogPriceOptions = {
     hideTaxLabel?: boolean; 
 }
```

Reply:
I would like to know the raison behind this change request :)

Reply:
to avoid the risk of interface duplication in Typescript, it is better to use type instead of  interface.